core/src/main/resources/error/error-classes.json

```diff
@@ -132,7 +132,97 @@
       },
       "INTERCEPTOR_RUNTIME_ERROR" : {
         "message" : [
-          "Error instantiating GRPC interceptor: <msg>"
+          "Error instantiating GRPC interceptor: <msg>."
+        ]
+      },
+      "INVALID_COMMAND_WRITE_INVALID_BUCKET_BY" : {
+        "message" : [
+          "BucketBy must specify a bucket count > 0, received <bucket> instead."
+        ]
+      },
+      "INVALID_PLAN_DEDUPLICATE_INVALID_INPUT" : {
+        "message" : [
+          "Deduplicate must be called with either all or a subset of column names to deduplicate."
+        ]
+      },
+      "INVALID_PLAN_DEDUPLICATE_INVALID_INPUT_COLUMN" : {
+        "message" : [
+          "Invalid column name used for Deduplicate: <col>."
+        ]
+      },
+      "INVALID_PLAN_EMPTY_INPUT" : {
+        "message" : [
+          "Input relation must not be empty."
+        ]
+      },
+      "INVALID_PLAN_INPUT" : {
+        "message" : [
+          "Error transforming Spark Connect plan: <msg>."
+        ]
+      },
+      "INVALID_PLAN_INVALID_ALIAS" : {
+        "message" : [
+          "Alias expressions with more than 1 identifier must not use optional metadata."
+        ]
+      },
+      "INVALID_PLAN_INVALID_DATA_SOURCE" : {
+        "message" : [
+          "Read from DataSource requires a format parameter."
+        ]
+      },
+      "INVALID_PLAN_INVALID_JOIN_CONDITION" : {
+        "message" : [
+          "Using columns or join conditions cannot be set at the same time in Join."
+        ]
+      },
+      "INVALID_PLAN_INVALID_SET_OPERATION" : {
+        "message" : [
+          "Invalid Set operation: <type> does not support <param>."
+        ]
+      },
+      "INVALID_PLAN_TRANSFORM_NA_FILL_EMPTY" : {
```

Comment:
Is it tested somewhere? If not, could add a test, please.

Reply:
Yes, there tests for all of these messages. Please see the SparkConnectPlannerSuite and SparkConnectProtoSuite.

Reply:
Done